rocAL/rocAL/include/loaders/circular_buffer.h

```diff
@@ -75,8 +75,8 @@ class CircularBuffer
     bool full();
     bool empty();
     size_t _buff_depth;
-    decoded_image_info _last_image_info;
-    std::queue<decoded_image_info> _circ_image_info;//!< Stores the loaded images names, decoded_width and decoded_height(data is stored in the _circ_buff)
+    decoded_sample_info _last_sample_info;
```

Comment:
Where is this _last_sample _info used?

Reply:
_circ_sample_info.push(_last_sample_info); - Its used to push in the _cic_sample_info